docs/api.md

```diff
@@ -10,6 +10,23 @@ nav_order: 3
 
 ## Class methods
 
+### .config
+
+
+
+### .config=(value)
+
+Sets the attribute config
```

Comment:
I agree that this could be changed. I figure it comes from `attr_accessor` as a default.

"Replaces the entire config. You shouldn't need to use this directly unless you're building a `ViewComponent::Config` elsewhere."